_docs/guides/integrating-vms.md

```diff
@@ -31,34 +31,41 @@ this infrastructure as a single mesh.
   [Installation guide]({{home}}/docs/setup/kubernetes/quick-start.html).
 
 * Deploy the [BookInfo]({{home}}/docs/guides/bookinfo.html) sample application.
-
-* Create a VM named 'db' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
+* Create a VM named 'mysqldb' in the same project as Istio cluster, and [Join the Mesh]({{home}}/docs/setup/kubernetes/mesh-expansion.html).
 
 ## Running mysql on the VM
 
 We will first install mysql on the VM, and configure it as a backend for the ratings service.
 
 On the VM:
 ```bash
-  sudo apt-get update && apt-get install ...
-  # TODO copy or link the istio/istio test script
-```
+  sudo apt-get update && sudo apt-get install -y mariadb-server
+  sudo mysql
+  # Grant remote access by root to db server
+  GRANT ALL PRIVILEGES ON *.* TO 'root'@'%' IDENTIFIED BY 'password' WITH GRANT OPTION;
+  quit;
+  # Allow remote access to mysql
+  sudo sed -i 's/^bind-address\(.*\)/#bind-address\1/g' /etc/mysql/mariadb.conf.d/50-server.cnf
+  sudo systemctl restart mysql
 
-## Registering the mysql service with the mesh
-
-### Machine admin
-First step is to configure the VM sidecar, by adding the service port and restarting the sidecar.
+```
+On the VM add ratings database to mysql. To make it easy to visually inspect the difference in the output of the bookinfo application, you can change the ratings that are generated by ratings service.
+```bash
+  # Add ratings db to the mysql db
+  curl -q https://raw.githubusercontent.com/istio/istio/master/samples/bookinfo/src/mysql/mysqldb-init.sql | sed -e 's/VALUES (\(.*\),.*/VALUES (\1, 1);/g' | sudo mysql
```

Comment:
what's the sed doing ? we can fix the script if there is an issue with it (which issue?)

Reply:
here i am changing the out put so that there is 1 star rating by each reviewer. Without this change it's really hard to know -visually-is the ratings db is generating the output or not.

Reply:
I prefer to do that with an update (that can be repeated)